api/python/t4/snapshots.py

```diff
@@ -308,6 +348,7 @@ def create_package(path):
 
             size = os.path.getsize(f)
             physical_keys = [{
+                'version': '0.0.1',
```

Comment:
Why do we need version on the physical_keys?

Reply:
It was part of the original proposed schema. I will drop it. We also decided to drop type from the physical keys for now

Reply:
look like this is still here. Leave type in for now, Dima's PRs have coded against it. We can rip out tomorrow later

Reply:
actually, we need version. Version ids are needed by boto, it doesn't handle query parameters correctly in the url so we need to explicitly

Reply:
I don't think this one was supposed to be the S3 version ID. In any case, I'll rebase my PR and add whatever is necessary.

Reply:
The version was always the physical_key schema version, not the s3 version or anything. I should have probably named it schema_version from the get-go